syft/workers/base.py

```diff
@@ -1004,9 +1024,7 @@ def message_pending_time(self, seconds: Union[int, float]) -> None:
         self._message_pending_time = seconds
 
     @staticmethod
-    def create_message_execute_command(
-        command_name: str, command_owner=None, return_ids=None, *args, **kwargs
-    ):
+    def create_message_execute_command(command_name: str, return_ids=None, *args, **kwargs):
```

Comment:
This method name is a bit confusing, it's very generic, while we are performing a specific ExecuteWorkerFunctionMessage order.
Maybe it could be renamed `create_message_execute_worker_function` or smthg like this?